src/Core/Silk.NET.Core/Loader/DefaultPathResolver.cs

```diff
@@ -276,14 +276,18 @@ private string GetUserDirectory()
         private List<string> GetAllRuntimeIds(string currentRid, DependencyContext ctx)
         {
             var allRiDs = new List<string>();
-            allRiDs.Add(currentRid);
-            if (!AddFallbacks(allRiDs, currentRid, ctx.RuntimeGraph))
+
+            if (ctx != null) // prevent null reference exception on net6.0-android where DependencyContext.Default is null
```

Comment:
Good catch! Just a small nitpick

```suggestion
            // prevent null reference exception on net6.0-android where DependencyContext.Default is null
            if (ctx is not null)
```

(we prefer `is not` as it can't be overloaded, but apparently this code is so old that it didn't follow that role elsewhere!)

Reply:
Done